src/Compilers/VisualBasic/Portable/Lowering/LocalRewriter/LocalRewriter_BinaryOperators.vb

```diff
@@ -84,20 +84,33 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         Public Overrides Function VisitBinaryOperator(node As BoundBinaryOperator) As BoundNode
             ' Do not blow the stack due to a deep recursion on the left. 
 
-            Dim child As BoundExpression = node.Left
+            Dim optimizeForConditionalBranch As Boolean = (node.OperatorKind And BinaryOperatorKind.OptimizableForConditionalBranch) <> 0
+            Dim optimizeChildForConditionalBranch As Boolean = optimizeForConditionalBranch
```

Comment:
>optimizeChildForConditionalBranch [](start = 16, length = 33)

Consider using `optimizeForConditionalBranch` directly.

Reply:
> Consider using optimizeForConditionalBranch directly.

It might have a different value, I would still need another local

---
In reply to: [328762339](https://github.com/dotnet/roslyn/pull/38802#discussion_r328762339) [](ancestors = 328762339)